doc/classes/FileAccess.xml

```diff
@@ -99,25 +99,25 @@
 				[b]Note:[/b] Only call [method flush] when you actually need it. Otherwise, it will decrease performance due to constant disk writes.
 			</description>
 		</method>
-		<method name="get_8" qualifiers="const">
+		<method name="get_8" qualifiers="const" deprecated="Use [method get_u8] or [method get_s8] instead.">
```

Comment:
Should the deprecated notice point to `get_u8` instead? This was the closest equivalent to it before.